src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsaccreditedprogrammesapi/restapi/controller/pni/response/model/NeedsScores.kt

```diff
@@ -0,0 +1,76 @@
+package uk.gov.justice.digital.hmpps.hmppsaccreditedprogrammesapi.restapi.controller.pni.response.model
+
+import com.fasterxml.jackson.annotation.JsonProperty
+import io.swagger.v3.oas.annotations.media.Schema
+
+/**
+ *
+ * @param sexScores
+ * @param cognitiveScores
+ * @param relationshipScores
+ * @param selfManagementScores
+ */
+data class NeedsScores(
+
+  @Schema(example = "null", description = "")
```

Comment:
either fill all these annotations in with something meaningful or remove the annotation